angular-route/templates/_.client.route.js

```diff
@@ -0,0 +1,5 @@
+$stateProvider.
+  state('<%= slugifiedName %>', {
+    url: '/<%= slugifiedRoutePath %>',
+    templateUrl: 'modules/<%= slugifiedModuleName %>/client/views/<%= slugifiedViewName %>.client.view.html'
+  }).
```

Comment:
Please, see `angular-route/index.js`, `renderRoute` function:
    
      renderRoute: function () {
        var routesFilePath = process.cwd() + '/modules/' + this.slugifiedModuleName + '/client/config/' + this.slugifiedModuleName + '.client.routes.js';
    
        // If routes file exists we add a new state otherwise we render a new one
        if (fs.existsSync(routesFilePath)) {
          // Read the source routes file content
          var routesFileContent = this.readFileAsString(routesFilePath);
    
          // Append the new state
          routesFileContent = routesFileContent.replace('$stateProvider.', this.engine(this.read('_.client.route.js'), this));
    
          // Save route file
          this.writeFileFromString(routesFileContent, routesFilePath);
        } else {
          this.template('_.client.routes.js', 'modules/' + this.slugifiedModuleName + '/client/config/' + this.slugifiedModuleName + '.client.routes.js')
        }
      }
    
The code is quite well commented, but I will repeat the idea here:
1. For module name `foo` check, that `foo.client.routes.js` already exists
2a. If it does, render the partial `_.client.route.js` and insert the rendered contents with new route description into an existing `foo.client.routes.js`
2b. If it does not exist, render template `_.client.routes.js` and save rendered result as `foo.client.routes.js`